.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
Please use our internal configuration as seen here: https://github.com/blumilksoftware/toby/blob/main/.eslintrc.js

Reply:
This configuration have problem with 
`
          const w = whitespaces[i] ?? '';
`

Reply:
all scripts included in blade would be called to file and included in blade

Reply:
> This configuration have problem with `const w = whitespaces[i] ?? '';`

What kind of problem?

> all scripts included in blade would be called to file and included in blade

I have no idea what you're trying to tell me.

Reply:
plugin of talwind have problem with js for a included a script a blade . I think it will be better to include all the scripts from blade to the js folder and include these files in the blade

Reply:
currently there is  js no in blade , so there is no point in letting the linter go there + your config refers to the talwind that you use in js files and we in blade, so it seems to me that it makes not sense to validate

Reply:
Please use our configuration. You can remove Tailwind rules if they're not necessary, but I want to have trailing commas forced and no semicolons in your JS codebase.

Reply:
okey i remove all talwind and vus. I left the basics rules from toby. It's okay ?